worlds/tunic/__init__.py

```diff
@@ -332,8 +370,73 @@ def remove_filler(amount: int) -> None:
             if tunic_item.name in slot_data_item_names:
                 self.slot_data_items.append(tunic_item)
 
+        # pull out the filler so that we can place it manually during pre_fill
+        self.fill_items = []
+        if self.options.local_fill > 0 and self.multiworld.players > 1:
+            # skip items marked local or non-local, let fill deal with them in its own way
+            # discard grass from non_local if it's meant to be limited
+            if self.settings.limit_grass_rando:
+                self.options.non_local_items.value.discard("Grass")
+            all_filler: List[TunicItem] = []
+            non_filler: List[TunicItem] = []
+            for tunic_item in tunic_items:
+                if (tunic_item.classification in [ItemClassification.filler, ItemClassification.trap]
```

Comment:
Not entirely equivalent, because of multi-classification items, but you could use the new thing:
```suggestion
                if (tunic_item.excludable
```

Reply:
We're not gonna make any prog+traps or useful+traps, so I'd be cool with that o:

Reply:
It might actually be equivalent. Not sure how `Flag in [Flag, Flag]` is precisely evaluated / if it covers multiclassifications. It might, but either way the new code would account for those if you did ever make them

Reply:
Yeah, the current one would only look for items that are purely traps or that are filler.